x-pack/plugins/monitoring/public/alerts/callout.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment } from 'react';
+import { EuiCallOut, EuiSpacer } from '@elastic/eui';
+import { CommonAlertStatus } from '../../common/types';
+import { AlertSeverity } from '../../common/enums';
+import { replaceTokens } from './lib/replace_tokens';
+import { AlertMessage } from '../../server/alerts/types';
+
+const TYPES = [
+  {
+    severity: AlertSeverity.Warning,
+    color: 'warning',
+    label: 'Warning alert(s)',
+  },
+  {
+    severity: AlertSeverity.Danger,
+    color: 'danger',
+    label: 'Danger alert(s)',
+  },
+];
+
+interface Props {
+  alerts: { [alertTypeId: string]: CommonAlertStatus };
+}
+export const AlertsCallout: React.FC<Props> = (props: Props) => {
+  const { alerts } = props;
+
+  const callouts = TYPES.map((type) => {
+    const list = [];
+    for (const alertTypeId of Object.keys(alerts)) {
+      const alertInstance = alerts[alertTypeId];
```

Comment:
Just curious why you chose to do it this way instead of the good ol `for in`':
```javascript
for (const alertTypeId in alerts) {
      const alertInstance = alerts[alertTypeId];
      ...
}
```

Reply:
Good question! No real reason I guess